dashboard/src/app.ts

```diff
@@ -0,0 +1,38 @@
+import express from "express";
+// This library allows us to change express's default behaviuor of handling 'async' errors
+// Instead of writing 'next(err)' we can directly 'throw' an error even inside 'async' function
+import "express-async-errors";
+import cors from "cors";
+import cookieSession from "cookie-session";
+
+import { brandRouter } from "./routes/brand";
+import { NotFoundError } from "common";
+import { errorHandler } from "common";
+
+const app = express();
+
+app.set("trust proxy", true);
+app.use(
+  cors({
+    origin: true,
+    credentials: true,
+  })
+);
+app.use(express.json());
+app.use(
+  cookieSession({
+    signed: false,
+    // disable secure cookies (transmitted only over https) for test environment
```

Comment:
**Issue**: The cookie session configuration disables secure cookies in production, which can expose sensitive session data. <br> **Fix**: Ensure that the 'secure' flag for cookies is set to true in production to prevent exposure of session identifiers. <br> **Code Suggestion**: 
 ```
 -    secure: process.env.NODE_ENV === "production",
 +    secure: true,
 ```